index.html

```diff
@@ -170,10 +170,10 @@ <h5 class="pointer">LET'S CONNECT</h5>
               <img src="./assets/angellist.svg" alt="Angellist" />
             </li>
           </ul>
-          <h5 class="btn">Get my resume</h5>
+          <a class="btn">Get my resume</a>
         </div>
-        <section id="lfs-section">
-          <div id="languages">
+        <ul id="lfs-section">
+          <li id="languages">
```

Comment:
<img width="789" alt="Screenshot 2022-07-01 at 02 10 33" src="https://user-images.githubusercontent.com/31113416/176793379-9e898b84-775c-43fa-9202-dea06ac6007e.png">

- [ ] As pointed out by the previous reviewer. unfortunately, this issue was not resolved with the latest changes.

Reply:
I already resolve it are you sure your opening the latest update?